src/midifile.c

```diff
@@ -443,7 +443,8 @@ static void read_playevent() {
 		switch (c) {
 		case 0xFF: /* meta event */
 			type = midigetc();
-			lookfor = midi->length_left - readvarinum();
```

Comment:
for whatever reason these lines were causing files to be parsed incorrectly, skipping over important system events like setting instruments and tempo